shared/feature/feeding/src/main/java/com/epmedu/animeal/feeding/presentation/viewmodel/handler/feeding/DefaultFeedingHandler.kt

```diff
@@ -85,6 +85,16 @@ class DefaultFeedingHandler(
             Cancel -> cancelFeeding()
             Expired -> expireFeeding()
             is Finish -> finishFeeding(event.feedingPhotos)
+            else -> restartFeedingState()
```

Comment:
I purpose to use the name of 'Reset' state here instead of 'else' - that will allow us to see all possible events just by looking as this handleFeedingEvent methods without guessing which events exist which we expect to see at "else" block and without opening Events enum. Plus, 'Reset' is a universal event fitting all possible cases when we want to start "restartFeedingState", so it's hard to imagine it would share this block with some other new event.